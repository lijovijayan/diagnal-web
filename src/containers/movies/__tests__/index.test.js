
import { Movies } from '../index';

describe('testing movies module', () => {
    it('rendered properly', () => {
        const movies = <Movies />
        expect(movies).toMatchSnapshot()
    })
})
