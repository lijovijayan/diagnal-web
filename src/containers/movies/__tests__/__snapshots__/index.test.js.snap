// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing movies module rendered properly 1`] = `<Movies />`;
